core/templates/pages/learner-group-pages/edit-group/edit-learner-group-page.component.html

```diff
@@ -0,0 +1,83 @@
+<div *ngIf="learnerGroup" class="oppia-edit-learner-group oppia-dashboard-view position-relative">
+  <div class="oppia-dashboard-view-tabs-container">
+    <div class="group-info">
+      <div class="group-info-title">
+        <span>{{learnerGroup.title}}</span>
+      </div>
+      <div class="group-info-description">
+        <span>
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="8" viewBox="0 0 16 8" fill="none">
+            <path d="M0 8V7.11667C0 6.68333 0.233333 6.33333 0.7 6.06667C1.16667 5.8 1.76667 5.66667 2.5 5.66667C2.64444 5.66667 2.77778 5.66944 2.9 5.675C3.02222 5.68056 3.14444 5.69444 3.26667 5.71667C3.17778 5.90556 3.11111 6.09722 3.06667 6.29167C3.02222 6.48611 3 6.69444 3 6.91667V8H0ZM4 8V6.91667C4 6.19444 4.36944 5.61111 5.10833 5.16667C5.84722 4.72222 6.81111 4.5 8 4.5C9.2 4.5 10.1667 4.72222 10.9 5.16667C11.6333 5.61111 12 6.19444 12 6.91667V8H4ZM13 8V6.91667C13 6.69444 12.9806 6.48611 12.9417 6.29167C12.9028 6.09722 12.8389 5.90556 12.75 5.71667C12.8722 5.69444 12.9944 5.68056 13.1167 5.675C13.2389 5.66944 13.3667 5.66667 13.5 5.66667C14.2556 5.66667 14.8611 5.8 15.3167 6.06667C15.7722 6.33333 16 6.68333 16 7.11667V8H13ZM8 5.5C7.11111 5.5 6.38889 5.63333 5.83333 5.9C5.27778 6.16667 5 6.50556 5 6.91667V7H11V6.9C11 6.5 10.725 6.16667 10.175 5.9C9.625 5.63333 8.9 5.5 8 5.5ZM2.5 5.16667C2.17778 5.16667 1.90278 5.05278 1.675 4.825C1.44722 4.59722 1.33333 4.32222 1.33333 4C1.33333 3.67778 1.44722 3.40278 1.675 3.175C1.90278 2.94722 2.17778 2.83333 2.5 2.83333C2.82222 2.83333 3.09722 2.94722 3.325 3.175C3.55278 3.40278 3.66667 3.67778 3.66667 4C3.66667 4.32222 3.55278 4.59722 3.325 4.825C3.09722 5.05278 2.82222 5.16667 2.5 5.16667ZM13.5 5.16667C13.1778 5.16667 12.9028 5.05278 12.675 4.825C12.4472 4.59722 12.3333 4.32222 12.3333 4C12.3333 3.67778 12.4472 3.40278 12.675 3.175C12.9028 2.94722 13.1778 2.83333 13.5 2.83333C13.8222 2.83333 14.0972 2.94722 14.325 3.175C14.5528 3.40278 14.6667 3.67778 14.6667 4C14.6667 4.32222 14.5528 4.59722 14.325 4.825C14.0972 5.05278 13.8222 5.16667 13.5 5.16667ZM8 4C7.44444 4 6.97222 3.80556 6.58333 3.41667C6.19444 3.02778 6 2.55556 6 2C6 1.43333 6.19444 0.958333 6.58333 0.575C6.97222 0.191667 7.44444 0 8 0C8.56667 0 9.04167 0.191667 9.425 0.575C9.80833 0.958333 10 1.43333 10 2C10 2.55556 9.80833 3.02778 9.425 3.41667C9.04167 3.80556 8.56667 4 8 4ZM8 1C7.72222 1 7.48611 1.09444 7.29167 1.28333C7.09722 1.47222 7 1.71111 7 2C7 2.27778 7.09722 2.51389 7.29167 2.70833C7.48611 2.90278 7.72222 3 8 3C8.28889 3 8.52778 2.90278 8.71667 2.70833C8.90556 2.51389 9 2.27778 9 2C9 1.71111 8.90556 1.47222 8.71667 1.28333C8.52778 1.09444 8.28889 1 8 1Z" fill="#757575"/>
```

Comment:
Oh, what is this for? Is there any way to clean this up?

Reply:
These are for svg icons of the tab, have tried to limit the max line size to make it look better. PTAL. Thanks!